integrations/vllm_plugin/vllm_tt/model_runner.py

```diff
@@ -1359,17 +1363,32 @@ def _dummy_run(self, num_tokens: int, num_reqs: int, num_blocks: int) -> None:
         if self.supports_mm_inputs:
             input_ids = None
             inputs_embeds = torch.zeros(
-                (num_tokens, self.hidden_size), dtype=self.dtype, device=self.device
-            )
+                (num_tokens, self.hidden_size),
+                dtype=self.dtype,
+            ).to(self.device)
         else:
-            input_ids = torch.zeros((num_tokens), dtype=torch.int32).to(self.device)
+            input_ids = torch.zeros(
+                (num_tokens,),
+                dtype=torch.int32,
+                device=self.device,
+            ).to(self.device)
```

Comment:
You are creating this tensor on device and then moving it again to device.
```suggestion
            input_ids = torch.zeros((num_tokens), dtype=torch.int32).to(self.device)
```

Reply:
I missed this one. Thanks